core/src/server/init/InstanceLockCheck.cpp

```diff
@@ -66,5 +73,11 @@ InstanceLockCheck::Check(const std::string& path) {
     return Status::OK();
 }
 
+void
+InstanceLockCheck::Release() {
+    auto fd = open(InstanceLockCheck::GetInstance()->lk_path.c_str(), O_RDWR | O_CREAT | O_NOFOLLOW, 0640);
```

Comment:
Need to check whether the lock file is openned successfully or not.

Reply:
Fix

Reply:
If fd is invalid, please also print the errno and error message, to indicate the error.